rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.adoc

```diff
@@ -1,7 +1,9 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="rosa-hcp-creating-cluster-with-aws-kms-key"]
 = Creating ROSA with HCP clusters using a custom AWS KMS encryption key
+
 include::_attributes/attributes-openshift-dedicated.adoc[]
+
```

Comment:
Sorry, I think I may have been confusing. Adding spaces to the headers _removes_ the topic map. These line breaks should be removed